src/__test__/Row.spec.js

```diff
@@ -39,4 +39,10 @@ describe('Row', () => {
     const row = container.querySelector('.row');
     expect(row.className).toBe('row row-cols-sm-2 row-cols-md-3');
   });
+
+  test('should bind inner div', () => {
+    let inner;
+    const { container } = render(Row, { props: { class: "test-123", inner }});
```

Comment:
Ah, it's because this is like:
```svelte
<Row class="test-123" inner />
```

but it should be:
```svelte
<Row class="test-123" bind:inner />
```

It doesn't seem to be possible to achieve this with ` @testing-library/svelte` (v3.0.3 as used here, and the latest available at time of writing) - though possibly it should be if it updated its version of [Svelte's component API](https://svelte.dev/docs#run-time-client-side-component-api-creating-a-component) - it's not clear to me what `context` is, it sounds like maybe `{ props: {...}, context: {"bind:inner": inner }` should work, but it claims that:

> Valid Svelte options are [anchor,props,hydrate,intro]

which is incomplete, per the link above.